ocs_ci/helpers/storageclass_helpers.py

```diff
@@ -0,0 +1,54 @@
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+
+
+def get_default_storage_class_name(interface, external_mode=False):
+    """
+    This Function will return the default storage class name from the cluster.
+    This will return only "cephfs", "rbd" and "rgw" type storage class names only.
+    Args:
+        interface (str): storage class interface type
+        external_mode (bool) : True if external mode setup else False.
+    Returns:
+        (str): storage class name
+    """
+
```

Comment:
please add an info log message `getting the default storageclass name`

Reply:
Added log message